torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -244,6 +256,18 @@ RegisterOperators reg(
            push(stack, (int)b);
            return 0;
          }),
+     Operator(
+         "prim::Int(Scalar a) -> float",
+         [](Stack& stack) {
+           IValue scalar;
```

Comment:
Popping these off as an `at::Scalar` instead would add the run-time assert that it has the right IValue tag

Reply:
Scalars can also be ints / floats